docs/spec-files/feature_discovery.md

```diff
@@ -85,7 +85,7 @@ A `discover-features/disclose` message looks like this:
             },
             {
                 "feature-type": "goal-code",
-                "id": "aries.sell.goods.consumer"
+                "id": "org.didcomm.sell.goods.consumer"
```

Comment:
These probably need to be converted into URI based identifiers if they're going to need to be standardized on and useful for the processing of messages without causing name collision issues. What you're suggesting could be the syntax of the URI we use if we want but we'll likely want to define the structure of this.

Reply:
I'm pretty resistant to going to that level. Experience in DIDComm v1 does not signal it to be a problem, and moving to non-conflicting values in the future is a non-breaking change.

Reply:
That would depend on the implementation. If I decide to only check that the value is a string and then the spec moves to URIs then you're correct it's not a breaking change. However, if I decide to use a regex to validate it's a reverse package notation and then a URI is used it would be an issue.

I'm not gonna die on this hill if you're adamant in not changing it, but I do think it's a mistake that will subtlety sneak up on us at scale and force us to build a centralized namespace registry to avoid conflicts here.

Reply:
@kdenhartog -- you are hereby granted full access to an "I-told-you-so" token to place in the collective face of the WG. You may play your token at any time.

:-)

Reply:
I've been waiting to earn one of those elusive tokens 😄